lib/matplotlib/legend.py

```diff
@@ -130,29 +131,31 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
     artists.  This option can be quite slow for plots with large amounts of
     data; your plotting speed may benefit from providing a specific location.
 
+    For back-compatibility, ``'center right'`` (but no other location) can also
+    be spelled ``'right'``, and each "string" locations can also be given as a
+    numeric value. Possible (case-sensitive) strings and codes are:
```

Comment:
New paragraph for "Possible ...". Backward-compatibility should be a paragraph of its own.